app/components/sidebar_controls_component.rb

```diff
@@ -70,7 +70,8 @@ def edit_collections
 
   def change_source_id
     render ActionButton.new(url: source_id_ui_item_path(id: pid),
-                            label: 'Change source id')
+                            label: 'Change source id',
+                            disabled: !item?)
```

Comment:
Do we want to show this button at all on non-item objects?

Reply:
So return from the method if its not an item, instead of disabling? 

Seems ok to me, though I'm not sure that's been our pattern in the past (where we instead turn buttons off if you can't do them).  What do you think @andrewjbtw ?

Reply:
There are many buttons that just don't show if this is not the right type of object. I don't think there is consistency in the way this has been done.  I think the disabled is confusing to people because they don't always know why it's disabled.

Reply:
ok, will defer to andrew's preference

Reply:
I agree with Justin. The buttons are already different for each item type and just leaving it out for collection seems fine.

I documented the existing blue button behavior before the work cycle as preparation for going to new page designs. It's definitely not consistent. We didn't really think about redesigning the sidebar as it already is since we were hoping to leave the buttons behind completely: https://consul.stanford.edu/pages/viewpage.action?pageId=163481988

Reply:
ok, updated the PR and tests to hide the button instead of disabling it

Reply:
ahh, missed the solr document setup means i don't need to stub the item method...made all of the changes requested